torchgeo/datasets/geo.py

```diff
@@ -706,7 +706,16 @@ def __getitem__(self, query: BoundingBox) -> dict[str, Any]:
             # with the default fill value and dtype used by rasterize
             masks = np.zeros((round(height), round(width)), dtype=np.uint8)
 
-        sample = {"mask": torch.tensor(masks), "crs": self.crs, "bbox": query}
+        # Convert masks from numpy to torch.
```

Comment:
I realized that this functionality is already present in https://github.com/microsoft/torchgeo/blob/1c20678a1001805412595be9759dae18a5d956df/torchgeo/datasets/geo.py#L547. Should I move the code to a separate function?

Reply:
Yes, this makes sense as a separate function. It can go in `torchgeo/datasets/utils.py`.